checks.d/http_check.py

```diff
@@ -262,6 +262,17 @@ def _check(self, instance):
             running_time = time.time() - start
             # Store tags in a temporary list so that we don't modify the global tags data structure
             tags_list = list(tags)
+
+            # Check for ill-formatted tags that aren't string types and log an error
+            bad_tags = []
+            for tag in tags_list:
+                if type(tag) is not str:
+                    bad_tags.append(tag)
+
+            if bad_tags:
+                self.log.error('Tags not formatted correctly for %s. %s not of type string'
```

Comment:
I think this log line will be more meaningful if we print one line per bad tag, in this case we'd print a single line for all bad tags, which might confuse the user.